enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -2231,6 +2238,92 @@ def test_can_redeem_subsidy_client_http_error(self, mock_get_client):
             'subsidy_status_code': str(status.HTTP_503_SERVICE_UNAVAILABLE),
         }
 
+    @ddt.data(
+        {'is_staff': True, 'lms_user_id_override': 1234},
+        {'is_staff': True, 'lms_user_id_override': None},
+        {'is_staff': False, 'lms_user_id_override': 5678},
+        {'is_staff': False, 'lms_user_id_override': None}
```

Comment:
[totally optional] Sometimes for clarity, it's nice to put `expected_can_redeem` as a ddt parameter in here just to make it super obvious what the expected output is for a given set of input. Like
```python
{'is_staff': True, 'lms_user_id_override': 1234, 'expected_can_redeem': True},
{'is_staff': True, 'lms_user_id_override': None, 'expected_can_redeem': False},
```